tests/api_workflow/test_api_workflow_download_dataset.py

```diff
@@ -52,6 +52,10 @@ def test_export_label_studio_tasks_by_tag_name(self):
         self.assertIsNotNone(tasks)
         self.assertTrue(all(isinstance(task, dict) for task in tasks))
 
+    def test_export_tag_to_basic_filenames_and_read_urls(self):
+        mappings = self.api_workflow_client.export_filenames_and_read_urls_by_tag_name('initial-tag')
+        self.assertIsNotNone(mappings)
+        self.assertTrue(all(isinstance(mapping, dict) for mapping in mappings))
```

Comment:
Rather assert that it `isinstance(..., FilenameAndReadURL)` for every element in `filenames_and_read_urls`